src/components/Commune/CommuneCertificationBar.tsx

```diff
@@ -0,0 +1,201 @@
+                    margin-top: 1rem;
+                    font-weight: bold;
+                }
+
+                label {
+                    font-size: 1.3rem;
+
+                    > i {
+                      margin-right: 0.5rem;
+                    }
+                }
+            }
+        }
+
+
+@media screen and (max-width: ${({ theme }) => theme.breakpoints.md}) {
+
+    > div {
+        .adresse-recap {
+            justify-content: flex-start;
+        }
+
+        .publication-recap {
+            padding: 0 1rem;
+
+            div {
+                background-color: white;
+                border-radius: 10px;
+                padding: 1rem;
+                margin-bottom: 1rem;
+                margin-top: 1rem;
+            }
+
+            label {
+                font-size: 1.3rem;
+            }
+        }
+    }
+}
+
+
+`
+
+export function CommuneCertificationBar({ commune, certificationPercentage, communeHasBAL, lastRevisionsDetails }: CommuneCertificationBarProps) {
+  return (
+    <StyledWrapper $certificationPercentage={certificationPercentage}>
+      <CardWrapper>
+        <div className="adresse-recap">
+          <div>
+            {commune.nbVoies}
+          </div>
+          <label>
+            voies, places et lieux-dits adressés
+          </label>
+        </div>
+        <div className="adresse-recap">
+          <div>
+            {commune.nbNumeros}
+          </div>
+          <label>
+            adresses
+          </label>
+        </div>
+        <div className="adresse-recap">
+          <div>
+            {certificationPercentage} %
+          </div>
+          <label>
+            d&apos;adresses certifiées
+          </label>
+        </div>
+      </CardWrapper>
+      <CardWrapper>
+        <div className="publication-recap">
+          <label>
+            <i className="fr-icon-send-plane-line" />
+            Mode de publication
+          </label>
+          <div>
+            {!communeHasBAL && 'Assemblage'}
+            {communeHasBAL && lastRevisionsDetails && lastRevisionsDetails[0][2]}
+          </div>
+        </div>
+        <div className="publication-recap">
+          <label>
+            <i className="ri-plug-line" />
+            Source
+          </label>
+          <div>
+            {!communeHasBAL && formatFr(assemblageSources(commune.voies))}
+            {communeHasBAL && lastRevisionsDetails && lastRevisionsDetails[0][3]}
+          </div>
+        </div>
+        <div className="publication-recap">
+          <label>
+            <i className="fr-icon-refresh-line" />
+            Dernière mise à jour
+          </label>
+          <div>
+            {!communeHasBAL && '-'}
+            {communeHasBAL && lastRevisionsDetails && (lastRevisionsDetails[0][1] as string).split(' à ')[0]}
+          </div>
+        </div>
+      </CardWrapper>
+      <CardWrapper>
+        <div className="publication-recap">
+          <label>
+            <i className="ri-key-line" />
```

Comment:
```suggestion
            <i className="ri-links-line" />
```

C'est l'icone qui est utilisé dans l'explorateur. La clé est utilisé pour le BanID

Reply:
ça marche